src/pages/dmt5/_dmt5-video.js

```diff
@@ -58,67 +72,64 @@ const Step = styled(Header)`
     }
 `
 
-class DtraderTabs extends React.Component {
-    my_ref = React.createRef()
-    interval_ref = undefined
-    state = {
-        current_step_image: 'dmt-5-step-1.png',
-        step: 1,
-    }
-    clickHandler = (image_name, step) => {
-        this.setState({ current_step_image: image_name, step })
-    }
-    render() {
-        return (
-            <Container>
-                <TabsWrapper>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            no_margin
-                            image_name="dmt-5-step-1.png"
-                            current_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dmt-5-step-1.png', 1)}
-                        >
-                            {localize('1. Sign up with Deriv')}
-                        </Step>
-                    </Tab>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dmt-5-step-2.png"
-                            current_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dmt-5-step-2.png', 2)}
-                        >
-                            {localize('2. Create a DMT5 account')}
-                        </Step>
-                    </Tab>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dmt-5-step-3.png"
-                            current_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dmt-5-step-3.png', 3)}
-                        >
-                            {localize('3. Fund your account')}
-                        </Step>
-                    </Tab>
-                </TabsWrapper>
-                <VideoWrapper>
-                    <Image
-                        img_name={this.state.current_step_image}
-                        alt={localize('DMT5 - Step') + this.state.step}
-                        width="100%"
-                    />
-                </VideoWrapper>
-            </Container>
-        )
+const DtraderTabs = () => {
+    const data = useStaticQuery(query)
+    const [current_step, setStep] = React.useState('step_1')
+    const clickHandler = (incoming_step) => {
+        if (incoming_step === current_step) return
```

Comment:
Like before, I think you don't need this check too.

Reply:
here too :pray: